tests/maskers/test_text.py

```diff
@@ -105,18 +105,22 @@ def test_keep_prefix_suffix_tokenizer_parsing():
     tokenizer_mt = AutoTokenizer.from_pretrained("Helsinki-NLP/opus-mt-en-es")
     tokenizer_gpt = AutoTokenizer.from_pretrained("gpt2")
     tokenizer_bart = AutoTokenizer.from_pretrained("sshleifer/distilbart-xsum-12-6")
+    tokenizer_mistral = AutoTokenizer.from_pretrained("mistralai/Mistral-7B-v0.1")
```

Comment:
Thanks for adding this to the test. This test  is skipped though, could we try to let it run in the CI and check whether it works? If not I am totally willing to revert to the current state, just would be great to increase the test coverage if possible.

Reply:
Locally the test works as expected.

Reply:
Yes, I also found that so I hope that it also runs in the CI.